### ISSUE TITLE
 RAI_ModelCreateTorch switch (device), Conditional jump or move depends on uninitialised value(s)

### ISSUE THREAD

**Issue by filipecosta90** [RedisAI/RedisAI#286]
commit e4f45f6268bb7a4b51d1170c28a590f6f628072e (HEAD -> coverage)
issue on
```
switch (device) {
    case RAI_DEVICE_CPU:
      dl_device = kDLCPU;
      break;
    case RAI_DEVICE_GPU:
      dl_device = kDLGPU;
      break;
    default:
      RAI_SetError(error, RAI_EMODELCONFIGURE, "Error configuring model: unsupported device\n");
      return NULL;
  }
```
```
==28935== Conditional jump or move depends on uninitialised value(s)
==28935==    at 0x6A7A040: RAI_ModelCreateTorch (torch.c:29)
==28935==    by 0x67E6B05: RAI_ModelCreate (model.c:209)
==28935==    by 0x67D784B: RedisAI_ModelSet_RedisCommand (redisai.c:620)
==28935==    by 0x1C76BE: RedisModuleCommandDispatcher (module.c:575)
==28935==    by 0x13D315: call (server.c:2468)
==28935==    by 0x13DFCA: processCommand (server.c:2763)
==28935==    by 0x14F086: processInputBuffer (networking.c:1470)
==28935==    by 0x14F1DB: processInputBufferAndReplicate (networking.c:1505)
==28935==    by 0x14F551: readQueryFromClient (networking.c:1587)
==28935==    by 0x1342E2: aeProcessEvents (ae.c:443)
==28935==    by 0x1344FC: aeMain (ae.c:501)
==28935==    by 0x141FD3: main (server.c:4234)
==28935== 
==28935== Conditional jump or move depends on uninitialised value(s)
==28935==    at 0x6A7A045: RAI_ModelCreateTorch (torch.c:29)
==28935==    by 0x67E6B05: RAI_ModelCreate (model.c:209)
==28935==    by 0x67D784B: RedisAI_ModelSet_RedisCommand (redisai.c:620)
==28935==    by 0x1C76BE: RedisModuleCommandDispatcher (module.c:575)
==28935==    by 0x13D315: call (server.c:2468)
==28935==    by 0x13DFCA: processCommand (server.c:2763)
==28935==    by 0x14F086: processInputBuffer (networking.c:1470)
==28935==    by 0x14F1DB: processInputBufferAndReplicate (networking.c:1505)
==28935==    by 0x14F551: readQueryFromClient (networking.c:1587)
==28935==    by 0x1342E2: aeProcessEvents (ae.c:443)
==28935==    by 0x1344FC: aeMain (ae.c:501)
==28935==    by 0x141FD3: main (server.c:4234)

```

**Comment by lantiga** [comment]
Fixed in #289 (see 47f7219)

**Comment by filipecosta90** [comment]
fixed :+1:
